packages/amplify-cli/src/extensions/amplify-helpers/resource-status.js

```diff
@@ -56,7 +56,30 @@ function capitalize(str) {
   return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
 }
 
-function getAllResources(amplifyMeta, category, resourceName) {
+function getCommonResources(resources, filteredResources) {
+  if (!filteredResources) {
+    return resources;
+  }
+
+  resources = resources.filter((resource) => {
+    let common = false;
+    for (let i = 0; i < filteredResources.length; i += 1) {
+      if (filteredResources[i].category === resource.category
```

Comment:
We could make (Array.Find)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find] to make this more readable